demo/video_demo.py

```diff
@@ -23,6 +23,16 @@ def parse_args():
         type=float,
         default=1,
         help='The interval of show (s), 0 is block')
+    parser.add_argument(
+        '--cat-id',
+        type=int,
```

Comment:
would it be better if it can be a list?

Reply:
I was being consistent with the way mmpose had an argument like that, but I agree that for mmdetection it would be better to be more flexible with the classes. My new commit addresses this (and I tested it).